src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
it's a mishmash of several rules we've been following. Like only declaring vars at the top and using function expressions instead of function declarations.